apps/drec-ui/src/apps/device-group/logic/detailView/detailView.ts

```diff
@@ -0,0 +1,72 @@
+import { EnergyTypeEnum, getEnergyTypeImage, getMainFuelType, PowerFormatter } from 'utils';
+import { TUseDeviceGroupDetailViewLogic } from './types';
+
+export const useDeviceGroupDetailViewLogic: TUseDeviceGroupDetailViewLogic = ({
+    deviceGroup,
+    owner,
+    allTypes
+}) => {
+    const locationProps = {
+        // @should be changed to actual owner name
+        owner: `DeviceGroup owner organization id ${owner}`,
```

Comment:
While designing the api here you can maybe tackle this issues? 
This comment originally was left for Origin Device API where device owner name was missing in schema

Reply:
Solution would be to bring to organization along with the device group but not sure if this is ok or not. Will keep in mind.